app/client/components/liveChat/liveChat.tsx

```diff
@@ -1,6 +1,6 @@
 import { css, SerializedStyles } from "@emotion/core";
 import { Button } from "@guardian/src-button";
-import { SvgArrowRightStraight } from "@guardian/src-icons/arrow-right-straight";
+import { SvgArrowRightStraight } from "@guardian/src-icons";
```

Comment:
umm, did this import work before?

Reply:
It worked fine in the app, although threw an error in Jest. (I guess because the Babel setup is different so it was expecting a pre-bundled import.)